templates/payment_success.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Registration Successful</title>
+    <link rel='stylesheet prefetch'
+        href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.css'>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <h2>You are registered for the DESC August 2022 collaboration meeting!</h2>
+        <br>
+
+        <div class="alert alert-info" role="alert">
+            You should have received by email a confirmation of your payment from <i>maureenlowery@uchicago.edu</i>. You
+            might notice that <b>the receipt will mention CMB Pol Workshop but this is normal</b>. If you have any
+            questions
+            regarding payment, please reach out to the LOC.
+
+        </div>
+
+        We are looking forward to seeing you at the meeting!<br>
+
+        <br>
+        The <a href="https://confluence.slac.stanford.edu/x/fQzUEw" target="_blank">Participants tab</a> on the
```

Comment:
Change link to point to Aug 2022 participants page.   See value in comment below